00_functions.R

```diff
@@ -361,23 +361,44 @@ is_hashed <- function() {
 
 # Non-Residential Long Stayers --------------------------------------------
 
-calculate_long_stayers <- function(input, projecttype){
+calculate_long_stayers <- function(days, projecttype){
+  
+  cls_df <- validation %>%
+    left_join(CurrentLivingSituation %>%
+                select(CurrentLivingSitID,
+                       EnrollmentID,
+                       InformationDate), by = "EnrollmentID") %>%
+    group_by(EnrollmentID) %>%
+    slice_max(InformationDate) %>%
+    ungroup() %>%
+    select(EnrollmentID, "MaxCLSInformationDate" = InformationDate)
```

Comment:
SUPER small suggestion: I think we could simplify this a bit. Since we're left-joining to the served_in_date_range dataset anyway, we don't need to start with validation. :

`cls_df <- CurrentLivingSituation %>%
    group_by(EnrollmentID) %>%
    slice_max(InformationDate) %>%
    ungroup() %>%
    select(EnrollmentID, "MaxCLSInformationDate" = InformationDate)`